modules/transactionpool.go

```diff
@@ -82,10 +82,6 @@ type TransactionPool interface {
 	// within 10 blocks.
 	FeeEstimation() (minimumRecommended, maximumRecommended types.Currency)
 
-	// IsStandardTransaction returns `err = nil` if the transaction is
-	// standard, otherwise it returns an error explaining what is not standard.
-	IsStandardTransaction(types.Transaction) error
-
```

Comment:
Woo, shrinking an interface! I guess we just never use this anywhere? Does it still need to be an exported method, then?